tests/app/helpers/test_path_finder_helper.py

```diff
@@ -0,0 +1,11 @@
+from unittest.mock import patch
+from tests.app.app_context_test_case import AppContextTestCase
+
+from app.helpers.path_finder_helper import path_finder
+
+@patch('app.helpers.path_finder_helper.get_metadata')
+@patch('app.helpers.path_finder_helper.get_answer_store')
+@patch('app.helpers.path_finder_helper.PathFinder')
+class TestPathFinderHelper(AppContextTestCase):
+    def test_path_finder_is_a_PathFinder(self, mock_metadata, mock_answer_store, mock_path_finder):
+        self.assertEqual('', mock_path_finder)
```

Comment:
This isn't very obvious?
looks like its asserting a mock is an empty string?

Reply:
We should also test that calling it a 2nd time returns the same value and doesn't generate a new one?